### ISSUE TITLE
FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#677]
I get the following warning when using Ginga:
```
...\ginga\AstroImage.py:183: FutureWarning: Using a non-tuple sequence
for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead
of `arr[seq]`. In the future this will be interpreted as an array index,
`arr[np.array(seq)]`, which will result either in an error or a different result.
  data = self.get_mddata()[view]
```

I think it is from numpy/numpy#9686 ?

**Comment by pllim** [comment]
1.15.0

**Comment by ejeschke** [comment]
@pllim, I remember also seeing this briefly with one miniconda install, but can't reproduce the error message now.  Does the message go away if you replace the line

    view = revnaxis + [slice(None), slice(None)]

with

    view = tuple(revnaxis + [slice(None), slice(None)])

?

**Comment by ejeschke** [comment]
That seems to fix it for me. You?  If so, I will submit a PR.

**Comment by ejeschke** [comment]
Found and fixed a place in `Cuts` that also had this issue.  Now checking `LineProfile`.

**Comment by ejeschke** [comment]
Fixed 2 spots in `LineProfile`.  That's all the instances of the problem I could find.

**Comment by pllim** [comment]
Looks like this is addressed in #680. Thanks!
